packages/worksheet/src/components/CreateWorksheet/WorksheetTemplate.js

```diff
@@ -0,0 +1,97 @@
+import { H1, H3, IconByName } from "@shiksha/common-lib";
+import { HStack, Stack, Button, Box, VStack, Pressable } from "native-base";
+import React from "react";
+import { useTranslation } from "react-i18next";
+import "react-responsive-carousel/lib/styles/carousel.min.css";
+import { Carousel } from "react-responsive-carousel";
+
+export default function WorksheetTemplate({ onPress }) {
+  const { t } = useTranslation();
+  const [selected, setSelectData] = React.useState();
+
+  const handleTemplateSelect = (value) => {
+    setSelectData(value);
+  };
+
+  return (
+    <Box>
+      <Box bg="worksheetCard.200" p="5">
+        <Carousel
+          showArrows={false}
+          showThumbs={false}
+          showStatus={false}
+          renderIndicator={(nextFun, value) => {
+            return (
+              <IconByName
+                onPress={(e) => nextFun()}
+                display="inline-block"
+                name={
+                  value ? "RecordCircleFillIcon" : "CheckboxBlankCircleLineIcon"
+                }
+                color="button.500"
+                p="1"
+              />
+            );
+          }}
+        >
+          {["", "", ""].map((item, index) => {
+            const color = selected === index ? "button.100" : "gray.200";
+            return (
+              <Pressable
+                key={index}
+                m="10px"
+                mb="10"
+                onPress={(e) => handleTemplateSelect(index)}
+              >
+                <Box bg="white" p="5" alignItems="center">
+                  <VStack w="100%" space="5">
+                    <HStack w="100%" space="5">
+                      <Box bg={color} w="44px" h="44px" />
+                      <Stack space="2" flex="1">
+                        <Box bg={color} h="13px" />
+                        <Box bg={color} h="22px" />
+                      </Stack>
+                    </HStack>
+                    {["", "", ""].map((subItem, index) => (
+                      <VStack key={index} w="100%" space="2">
+                        <Box bg={color} w="18px" h="10px" />
+                        <Box bg={color} h="32px" />
+                      </VStack>
+                    ))}
+                    <VStack w="100%" space="2">
+                      <Box bg={color} h="10px" w="25%" />
+                      {["", "", ""].map((subItem, index) => (
+                        <HStack key={index} space="4">
+                          <Box bg={color} h="10px" flex="1" />
+                          <Box bg={color} h="10px" flex="1" />
+                        </HStack>
+                      ))}
+                    </VStack>
+                  </VStack>
+                </Box>
+                <VStack p="5">
+                  <H1>Template No. {index + 1}</H1>
```

Comment:
translation required

Reply:
as of now this is static data and after its coming from API so we can't implement  translation